web/client/codechecker_client/cmd_line_client.py

```diff
@@ -1139,6 +1143,48 @@ def print_gerrit_json(reports, output_dir):
         LOG.info("Gerrit review file was created: %s\n",
                  gerrit_review_json)
 
+    def write_codeclimate_json(reports, output_dir):
+        """ Converts the given reports to codeclimate format.
+
+        This function will convert the given report to Code Climate format
+        and writes the issues to the given output directory to a JSON file.
+        """
+        repo_dir = os.environ.get('CC_REPO_DIR')
+
+        issues = []
+        for report in reports:
+            if isinstance(report, Report):
```

Comment:
If it is not an instance of `Report` then we should build this `Report` object so `report.to_codeclimate()` can be used for the output conversion. This way this body of `to_codeclimate()` doesn't need to be duplicated a few lines below.